api/lib/application/organizations/index.js

```diff
@@ -540,6 +540,30 @@ exports.register = async (server) => {
         ],
       },
     },
+    {
+      method: 'DELETE',
+      path: '/api/organizations/{id}/invitations/{invitationId}/cancel',
+      config: {
+        pre: [
+          {
+            method: securityPreHandlers.checkUserIsAdminInOrganization,
+            assign: 'isAdminInOrganization',
+          },
+        ],
+        validate: {
+          params: Joi.object({
+            id: identifiersType.organizationId,
+            invitationId: identifiersType.organizationInvitationId,
```

Comment:
```suggestion
            organizationInvitationId: identifiersType.organizationInvitationId,
```

autant garder le même nommage partout et surtout on est plus explicite avec l'ajout du organization

Reply:
done :D